app/v1/models/response/listPeriodSummaries/PeriodDetails.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package v1.models.response.listPeriodic
+package v1.models.response.listPeriodSummaries
 
 import play.api.libs.json.{JsPath, Json, OWrites, Reads}
 import play.api.libs.functional.syntax._
```

Comment:
This is probably a bit picky, but should the transaction reference be read directly from the downstream response rather than constructed as "from_to"? Just thinking that if downstream change their ID format later, this assumption would suddenly break.  Unlikely but kind of possible?